### ISSUE TITLE
Does this support the audience parameter?

### ISSUE THREAD

**Issue by adjenks** [RiskioFr/oauth2-auth0#18]
Does this support the audience parameter?
https://auth0.com/docs/flows/call-your-api-using-the-client-credentials-flow#parameters

Where do I put it?

**Comment by phmLabs** [comment]
Did you solve it?

**Comment by adjenks** [comment]
@phmLabs No, I think I just made the connection "by hand" using curl. I didn't use any library.
